tests/integration/suite/scheduler/quorum/notls.go

```diff
@@ -70,9 +73,14 @@ func (n *notls) Setup(t *testing.T) []framework.Option {
 		scheduler.New(t, append(opts, scheduler.WithID("scheduler2"), scheduler.WithEtcdClientPorts(clientPorts))...),
 	}
 
+	schedulerAddresses := []string{n.schedulers[0].Address(), n.schedulers[1].Address(), n.schedulers[2].Address()}
+	n.daprd = daprd.New(t,
+		daprd.WithSchedulerAddresses(strings.Join(schedulerAddresses, ",")),
```

Comment:
I updated it to pass in the addresses separately, like:
```
daprd.WithSchedulerAddresses(s.schedulers[0].Address(), s.schedulers[1].Address(), s.schedulers[2].Address()),
```

I see what you were saying, that I didn't need to do it here since its already done in the framework.